ova/Libraries/provision-opendistro.sh

```diff
@@ -192,22 +182,19 @@ installElasticsearch() {
 installFilebeat() {
     logger "Installing Filebeat..."
 
-    #eval "echo -e '[elasticsearch-7.x]\nname=Elasticsearch repository for 7.x packages\nbaseurl=https://artifacts.elastic.co/packages/7.x/yum\ngpgcheck=1\ngpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch\nenabled=1\nautorefresh=1\ntype=rpm-md' | tee /etc/yum.repos.d/elastic.repo"
-
-    eval "$sys_type install filebeat-"${ELK_VERSION}" -y -q  $debug"
+    $sys_type install filebeat-"${ELK_VERSION}" -y -q  $debug
     if [  "$?" != 0  ]
     then
         echo "Error: Filebeat installation failed"
         exit 1;
     else
-        eval "curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
-        eval "chmod go+r /etc/filebeat/wazuh-template.json $debug"
-        eval "curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module $debug"
-        eval "mkdir /etc/filebeat/certs $debug"
-        eval "cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ $debug"
-        ls mv /etc/elasticsearch/certs/
-        eval "mv /etc/elasticsearch/certs/filebeat* /etc/filebeat/certs/ $debug"
+        curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug
+        curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug
+        chmod go+r /etc/filebeat/wazuh-template.json $debug
+        curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module $debug
+        mkdir /etc/filebeat/certs $debug
```

Comment:
```suggestion
        mkdir -p /etc/filebeat/certs $debug
```

Reply:
Done in 52fa9adacc80d59386c2d0ed83176a4e11430553